db/seeds.rb

```diff
@@ -14,18 +14,18 @@ def yes_no(prompt)
 
 def create_restaurant(args = {})
   suffix = %W(Place Parlor House Bar Food\ Court Pub)
-  prefix = %w(The Local Homemade)
-  cat = %w(Italian Thai Contemporary).sample
+  prefix = %w(The Local Homemade Fancy Skitbra)
+  cat = %w(Italian Thai Pizza Vegan).sample
   name = "#{prefix.sample} #{cat} #{suffix.sample}"
   description = args[:description] || Faker::Lorem.paragraph(2)
   address = args[:address] || [Faker::Address.street_address, Faker::Address.postcode, Faker::Address.city].join(', ')
-  lat = rand(57.7089001..57.7090009).round(7)
-  lon = rand(11.9800001..11.9900009).round(7)
+  lat = rand(57.3149..57.7090009).round(7)
+  lon = rand(11.4911..11.9900009).round(7)
   rest = Restaurant.new(name: name,
                         description: description,
-                        address: address,
-                        cuisine: cat,
-                        user: User.all.sample,
+                        # zipcode: address,
+                        category: cat,
+                        # user: User.all.sample,
```

Comment:
Yeah I was surprised but apparently not. Because there are no validations? Anyway, doesn't throw any errors.